data/service/api/v1/summary.go

```diff
@@ -79,6 +85,109 @@ func GetSummary[T types.Stats, A types.StatsPt[T]](dataServiceContext dataServic
 	}
 }
 
+type RealtimePatientsResponse struct {
+	Config  RealtimePatientConfigResponse `json:"config"`
+	Results []RealtimePatientResponse     `json:"results"`
+}
+
+type RealtimePatientConfigResponse struct {
+	SchemaVersion int       `json:"schemaVersion"`
+	Code          string    `json:"code"`
+	ClinicId      string    `json:"clinicId"`
+	StartDate     time.Time `json:"startDate"`
+	EndDate       time.Time `json:"endDate"`
+}
+
+type RealtimePatientResponse struct {
+	Id               string    `json:"id"`
+	FullName         string    `json:"fullName"`
+	BirthDate        time.Time `json:"birthDate"`
+	MRN              *string   `json:"mrn"`
+	RealtimeDays     int       `json:"realtimeDays"`
+	IsSufficientData bool      `json:"isSufficientData"`
```

Comment:
Should this be `hasSufficientData`?

Reply:
was copied from the response json provided by frontend in the ticket, but it is a bit weird sounding, ill change it.